.travis.yml

```diff
@@ -5,8 +5,14 @@ notifications:
     on_success: never
     on_failure: change
 
+env:
+  global:
+    - CXX=g++-4.8
+
 node_js:
-  - 0.10
+  - stable
+  - v4.2
+  - v0.10
```

Comment:
I mostly left it in for legacy reasons, so no concerns about removing it. Want me to keep `stable`? For the time being `stable` is aliased to `4.2`, so we're building against the same version twice. I don't think stable will follow along with Argon releases (v4.2.x), assuming I understand their LTS system correctly — I think we should stick with LTS for the time being to minimize our own effort keeping this thing running on the "bleeding edge" releases.